app/views/support/schools/show.html.erb

```diff
@@ -24,7 +24,7 @@
 <h2 class="govuk-heading-l govuk-!-margin-top-9">Users</h2>
 
 <% if policy(User).new? %>
-  <%= govuk_button_link_to t('page_titles.new_school_user'), new_support_school_user_path(school_urn: @school.urn) %>
+  <%= govuk_button_link_to 'Invite a new user', new_support_school_user_path(school_urn: @school.urn) %>
```

Comment:
could user `t('page_titles.support.users.new')` instead of hard coded copy

Reply:
I thought about this, but we generally haven't been doing this in other places as there's quite a few examples where the link doesn't match the page title. So far, the I18n file has consistently been used for page titles and error messages but is patchy for anything else.